runtime/lua/vim/_defaults.lua

```diff
@@ -32,6 +32,11 @@ do
     nargs = 1,
     complete = 'file',
   })
+
+  vim.api.nvim_create_user_command('LspLog', function()
+    local log_path = vim.lsp.get_log_path()
+    vim.cmd(string.format('tabnew %s', log_path))
```

Comment:
Using `tabnew` here to [keep the same behavior as `nvim-lspconfig`](https://github.com/neovim/nvim-lspconfig/blob/61e5109c8cf24807e4ae29813a3a82b31821dd45/plugin/lspconfig.lua#L234), but I'm open to changing it.